anvio/pfam.py

```diff
@@ -163,8 +189,32 @@ def __init__(self, args, run=run, progress=progress):
 
 
     def is_database_exists(self):
-        if not os.path.exists(os.path.join(self.pfam_data_dir, 'Pfam-A.hmm.gz')):
+        """
+        This function verifies that pfam_data_dir contains the Pfam hmm profiles and checks whether they are compressed or not.
+        If they are compressed, we decompress them and run hmmpress.
+
+        PARAMETERS: N/A
+
+        RETURNS: N/A
```

Comment:
there is no need for these since the fact that the function does not take any parameters is obvious from the header :)